Edge feature on Windows 10 results in linker-error

I tried to run the basic todo example with the edge feature enabled:

```toml
webview-sys = { path = "webview-sys", version = "0.3.1", features=["edge"] }
```

```
cargo run --example todo
```

This resulted in a linker error:

> error LNK2019: Unresolved external symbol \"__imp_CoCreateInstance\" in function \"webview_dialog\".\r\nlibwebview_sys-573306a327093e04.rlib(webview_edge.o) : error LNK2019: Unresolved external symbol \"__imp_CoWaitForMultipleHandles\" in function \"\"public: __cdecl webview::webview::webview(void (__cdecl*)(void *,char const *),char const *,int,int,bool,bool)\" (??0webview@0@QEAA@P6AXPEAXPEBD@Z1HH_N3@Z)\".\r\n

The symbols are from Ole32.lib and I figured webview-sys should link against that. I had a look at webview-sys's build.rs and noticed that Ole32 is in fact not linked against if the edge feature is set. Adding `println!("cargo:rustc-link-lib=ole32");` solved my linker issue.

Has anyone else had this issue? If so, I propose to change `webview-sys/build.rs` accordingly. If not, I wonder why this linker-error occurs.

Some info:

- cargo 1.39.0 (1c6ec66d5 2019-09-30)
- rustup 1.20.2 (13979c968 2019-10-16)
- Default host: x86_64-pc-windows-msvc
- Default Toolchain: stable-x86_64-pc-windows-msvc
- VC++ 2017 version 15.9 v14.6 latest v141 tools
- Windows 10 SDK 10.0.17763.0

--- a/webview-sys/build.rs
+++ b/webview-sys/build.rs
@@ -26,10 +26,14 @@ fn main() {
             build
                 .file("webview_edge.cpp")
                 .flag_if_supported("/std:c++17");
+
+            for &lib in &["windowsapp", "user32", "gdi32", "ole32"] {
+                println!("cargo:rustc-link-lib={}", lib);
+            }
         } else {
             build.file("webview_mshtml.c");
 
-            for &lib in &["ole32", "comctl32", "oleaut32", "uuid", "gdi32"] {
+            for &lib in &["ole32", "comctl32", "oleaut32", "uuid", "gdi32", "user32"] {
                 println!("cargo:rustc-link-lib={}", lib);
             }
         }

--- a/webview-sys/webview_edge.cpp
+++ b/webview-sys/webview_edge.cpp
@@ -15,9 +15,10 @@
 #include <winrt/Windows.Foundation.h>
 #include <winrt/Windows.Web.UI.Interop.h>
 
-#pragma comment(lib, "windowsapp")
+#pragma comment(lib, "windowsapp.lib")
 #pragma comment(lib, "user32.lib")
-#pragma comment(lib, "gdi32")
+#pragma comment(lib, "gdi32.lib")
+#pragma comment(lib, "ole32.lib")
 
 static inline BSTR webview_to_bstr(const char *s) {
   DWORD size = MultiByteToWideChar(CP_UTF8, 0, s, -1, 0, 0);

--- a/webview-sys/webview_mshtml.c
+++ b/webview-sys/webview_mshtml.c
@@ -70,9 +70,12 @@ WEBVIEW_API webview_t webview_new(const char* title, const char* url, int width,
 	return wv;
 }
 
-#pragma comment(lib, "user32.lib")
 #pragma comment(lib, "ole32.lib")
+#pragma comment(lib, "comctl32.lib")
 #pragma comment(lib, "oleaut32.lib")
+#pragma comment(lib, "uuid.lib")
+#pragma comment(lib, "gdi32.lib")
+#pragma comment(lib, "user32.lib")
 
 #define WM_WEBVIEW_DISPATCH (WM_APP + 1)
 

